### DISCUSSION TITLE
Step form disappearing from screen

### DISCUSSION THREAD

**Discussion by edrisi6** [Budibase/budibase#8272] [Help] [answered]
and shows again once the component edit is opened and the step is selected

https://user-images.githubusercontent.com/4608742/196026233-3bcc8bcf-756f-4028-8c36-c0f7e4a3affc.mov

**Comment by melohagan** [comment, marked as answer]
Hey @edrisi6 

As mentioned in the issue you raised (https://github.com/Budibase/budibase/issues/8275) this is expected behaviour.

**Comment by edrisi6** [comment]
Great Thank you. On another matter, I am trying to use budibase forms to update various tables through one public form. Each form step updates a certain table. I could not figure out a way to jump from one form to the other without using too many buttons. Even then, I could not get things to work properly. From where could I get help if I may Ask?

**Reply by melohagan** [reply]
Hey @edrisi6 

You may have seen the docs for form steps already: https://docs.budibase.com/docs/form-steps

How many buttons are too many? In my mind you would have the following buttons in a typical form step flow:
1. Next button
2. Next and back buttons
3. Submit and back buttons

**Reply by edrisi6** [reply]
Well, it seems like the button only responds if it was under a Form component. And since each form component can update only one table, I have to use a few of them. Therefore ended up with the same Next buttons a few times (under each form component a button with a jump to specific step action) 

Please consider that I have close to zero database/development knowledge and I have only used Budibase for a few weeks now. Therefore my comprehension of the logical process is still limited

**Reply by melohagan** [reply]
No worries, I think instead of having multiple form components, the idea of forms steps is to have a single form.
Your tree would look something like:

> Form
> > Form Step 1
> > Form Step 2
> > Form Step 3

Then if you are saving to many tables, your [form schema](https://docs.budibase.com/docs/forms#form-schema) would be **custom**, and at each step you would save to the desired table, but as your schema is custom, you would need to manually *Add columns* using the `Form.Fields` bindings like in this screenshot:
![image](https://user-images.githubusercontent.com/101575380/196624749-f4c22af5-ab12-486f-853f-88067f1cbbd4.png)

**Reply by edrisi6** [reply]
Oh man, I almost lost an eye browsing the web for this tip. Thank you

**Reply by melohagan** [reply]
You're welcome! The docs section on it is just a single paragraph without screenshots: https://docs.budibase.com/docs/forms#custom-inputs--fields

I'll need to update that for sure! Although you may find the video at the bottom helpful!
